ideascube/configuration/registry.py

```diff
@@ -12,19 +12,29 @@
 REGISTRY = {
     # 'namespace': {
     #     'key': {
+    #         'summary': 'Help text for the management command',
+    #         'pretty_type': 'Pretty type(s) for the management command',
     #         'type': str,
     #         'default': 'Default value',
     #     },
     # },
     'server': {
         'site-name': {
+            'summary': 'The pretty name of the server, as seen by users of '
+                       'the web interface.',
+            'pretty_type': 'A string',
             'type': str,
             'default': getattr(settings, 'IDEASCUBE_NAME', 'Ideas Cube'),
         },
     },
 }
 
 
+def get_all_namespaces():
+    for namespace in sorted(REGISTRY):
+        yield namespace
+
```

Comment:
> It was a suggestion to use yield from sorted(REGISTRY) instead of the for loop.

Wait, that works? :open_mouth: 

[goes to try it out...]

Wow! :open_mouth: 

> The only "massive" use of yield from I know about was with asyncio before await came into python.

Yeah, coroutine stuff (not necessarily async) is the only place I've ever seen `yield from` being used.

I'm going to say no here, due to how rarely this seems to be. Familiarity isn't a trump-all card, but in this case, I'm going to stand by it.

Thanks for teacking me something, though!